src/matchers/utils.ts

```diff
@@ -113,4 +113,26 @@ export const getElementText = async (
   throw new Error(`Invalid input length: ${args.length}`)
 }
 
-export const quote = (val: string | null) => `'${val}'`
+export const quote = (val: string | null) =>
```

Comment:
do we really have a case when we will pass `null` here?

Reply:
I believe I saw this happen once during some testing, but I can't remember if that was just misconfiguration on my part.  That said, the type signature allows nulls so theoretically yes it could accept nulls.

Reply:
Maybe it will be better to return empty string for this case?

Reply:
Updated